src/data-parsing/info-parser.js

```diff
@@ -28,26 +28,29 @@ function parseEnumQuestion (question, chartRecipe) {
   Object.keys(parsedData).forEach(option => arr.push(parsedData[option]))
 
   chartRecipe.data = arr
-  chartRecipe.type = 'doughnut'
+  if (isBoolean) {
```

Comment:
What `isBoolean`? A bit unclear IMO :P

Reply:
It's not checking anything. It is instead of making two exact same functions for Boolean and Enum, I made one.
In the configuration file, added in  the last commit (cuz I was dumb and didn't add it in the first), Enum and Boolean can separate display configurations.

Reply:
@YonatanNitzan ah. kden. seems good